src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -516,7 +516,7 @@ public IClassificationLoss CreateComponent(IHostEnvironment env)
         /// ]]>
         /// </format>
         /// </example>
-        public static LogisticRegressionBinaryTrainer LogisticRegression(this BinaryClassificationCatalog.BinaryClassificationTrainers catalog,
+        public static LbfgsLogisticRegressionTrainer LbfgsLogisticRegressio(this BinaryClassificationCatalog.BinaryClassificationTrainers catalog,
```

Comment:
>LbfgsLogisticRegressio [](start = 53, length = 22)

you miss `n` in the end.
And I don't think this new verb any better than previous one.
More importantly I don't understand why you even changing this? We had @agoswami issue and PR regarding names, where we spend quite a lot of effort in attempt to figure out names, why you change it out of blue now?

Reply:
I don't think my changes violate his rule.

---
In reply to: [267478532](https://github.com/dotnet/machinelearning/pull/3034#discussion_r267478532) [](ancestors = 267478532)

Reply:
This PR makes me nervous so close to ship date. Let's get consensus in #3016 before we make the change.

---
In reply to: [267481247](https://github.com/dotnet/machinelearning/pull/3034#discussion_r267481247) [](ancestors = 267481247,267478532)